backend/sirius-web-compatibility/src/main/java/org/eclipse/sirius/web/compat/services/ExplorerTreeDescriptionProvider.java

```diff
@@ -64,6 +67,7 @@ public TreeDescription getTreeDescription() {
                 .elementsProvider(this::getElements)
                 .hasChildrenProvider(this::hasChildren)
                 .childrenProvider(this::getChildren)
+                .canCreatePredicate(variableManager -> false)
```

Comment:
- Representations need to indicate that they can be created
- The tree of the explorer will always be created

So I have chosen to create an predicate (which returns false) since I know that it will never be called to create the explorer and I don't want to let anyone create a tree representation with the description used for the explorer.

Reply:
I have added some documentation to make it more explicit